### PR TITLE
Add note on tempdb

### PR DESCRIPTION
**Pull request by way0utwest**
Tempdb is a system db, but it is encrypted with TDE.

### CODE REVIEW

**Comment by Court72** [comment]
@GithubMirek 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/database/transparent-data-encryption-tde-overview.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ For Azure SQL Database and Azure Synapse, the TDE protector is set at the [serve
...
 
 > [!NOTE]
-> TDE cannot be used to encrypt system databases, such as the **master** database, in Azure SQL Database and Azure SQL Managed Instance. The **master** database contains objects that are needed to perform the TDE operations on the user databases. It is recommended to not store any sensitive data in the system databases. [Infrastructure encryption](transparent-data-encryption-byok-overview.md#doubleencryption) is now being rolled out which encrypts the system databases including master. 
+> TDE cannot be used to encrypt system databases, such as the **master** database, in Azure SQL Database and Azure SQL Managed Instance. The **master** database contains objects that are needed to perform the TDE operations on the user databases. It is recommended to not store any sensitive data in the system databases. [Infrastructure encryption](transparent-data-encryption-byok-overview.md#doubleencryption) is now being rolled out which encrypts the system databases including master. The exception is that tempdb is encrypted with TDE whenever it is applied to any user database.
```
Comment (MashaMSFT) [inline azure-sql/database/transparent-data-encryption-tde-overview.md:34]:
```suggestion
> TDE cannot be used to encrypt system databases, such as the `master` database in Azure SQL Database and Azure SQL Managed Instance. The `master` database contains objects that are needed to perform TDE operations on user databases. It is recommended not to store any sensitive data in system databases.  The exception is `tempdb`, which is always encrypted with TDE to protect data stored there. 
```

**Comment by MashaMSFT** [comment]
#sign-off @court72 fyi

**Comment by MashaMSFT** [comment]
Thanks so much for your contribution @way0utwest! I had a chat with the PG and we modified the language slightly :) However, your changes should be life later this week. Thanks again for helping improve our documentation!

**Pull request merged**